### PR TITLE
Add option to ignore SSL issues

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

**Comment by scoheb** [comment]
@ggallen Please review

File: src/main/java/com/redhat/jenkins/plugins/cachet/ResourceUpdater.java [hunk 1]
```diff
@@ -87,6 +142,15 @@ public static void setResources() {
...
                 rmap = null;
+            } catch (NoSuchAlgorithmException e) {
+                log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
+                rmap = null;
+            } catch (KeyStoreException e) {
+                log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
+                rmap = null;
+            } catch (KeyManagementException e) {
+                log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
+                rmap = null;
```
Comment (ggallen) [inline src/main/java/com/redhat/jenkins/plugins/cachet/ResourceUpdater.java]: Why not just catch these all on the same line, like this?
    catch (IOException | NoSuchAlgorithmException | KeyStoreException | KeyManagementException e)
Then we can get rid of all the duplicated code....

File: src/main/java/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration.java [hunk 2]
```diff
@@ -39,6 +39,16 @@
 public final class GlobalCachetConfiguration extends GlobalConfiguration {
 
     private String cachetUrl;
+    private Boolean ignoreSSL;
```
Comment (ggallen) [inline src/main/java/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration.java]: Shouldn't this have a default value of false somewhere?  Or maybe BooleanUtils does this?

File: src/main/java/com/redhat/jenkins/plugins/cachet/ResourceUpdater.java (hunk 1 above)
Comment (scoheb) [reply src/main/java/com/redhat/jenkins/plugins/cachet/ResourceUpdater.java]: Yep. I will fix this mess.

File: src/main/java/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration.java (hunk 2 above)
Comment (scoheb) [reply src/main/java/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration.java]: I use BooleanUtils on another spot to handle the default...want me to handle it on the get() too?

**Pull request merged**